cl/corpus_importer/management/commands/make_aws_manifest_files.py

```diff
@@ -44,6 +45,10 @@
     if value == "Judge"
 ]
 
+BUCKET_ARN = "arn:aws:s3:::bulk-customer-data"
+
+ROLE_ARN = "arn:aws:iam::968642441645:role/bulk-customer-data-batch"
```

Comment:
Would it be a good idea to pass these as command arguments, or is it unlikely that they'll change in the future?

Reply:
It's unlikely the role ARN will change, but we can compute the bucket ARN since we're already passing the bucket name.